src/pkg/api/handlers.go

```diff
@@ -136,6 +136,39 @@ func getPod(cache *resources.Cache) func(w http.ResponseWriter, r *http.Request)
 	return rest.Bind(cache.Pods)
 }
 
+// @Description Get Pod logs
+// @Tags workloads
+// @Accept  html
+// @Produce  json
+// @Success 200
+// @Router /api/v1/resources/workloads/pods/logs [get]
+// @Param pod query string true "Pod name"
+// @Param namespace query string true "Namespace"
+func getPodLogs(k8sSession *session.K8sSession) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// todo: guard against accessing logs for pods that shouldn't be accessed?
+		// todo: test back/forward button, etc
+		pod := r.URL.Query().Get("pod")
+		namespace := r.URL.Query().Get("namespace")
+		if pod == "" || namespace == "" {
+			http.Error(w, "Pod and Namespace are required", http.StatusBadRequest)
+			return
+		}
+
+		// return fake logs for now
+		logs := []string{
+			"2024-01-01T00:00:00Z [INFO] This is a log message",
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		err := json.NewEncoder(w).Encode(logs)
+		if err != nil {
+			return
+		}
```

Comment:
If there's an error we should set the header status. 

```suggestion
		err := json.NewEncoder(w).Encode(logs)
		if err != nil {
                        slog.Error("failed to encode getPodLog response", "error", err)
		        http.Error(w, "Internal Server Error", http.StatusInternalServerError)
			return
		}
```
Note that if it's successful we don't need to send `StatusOK`, it's implied on the first write:

>  If WriteHeader is not called explicitly, the first call to Write will trigger an implicit WriteHeader(http.StatusOK).

- https://pkg.go.dev/net/http#pkg-types